app/models/policy_class.rb

```diff
@@ -42,6 +44,22 @@ def ==(other)
     end
   end
 
+  def previous
+    planning_application.policy_classes.where("section < ?", section).last
+  end
+
+  def next
+    planning_application.policy_classes.where("section > ?", section).first
+  end
+
+  def default_path
```

Comment:
Well, given that a model is meant to be a representation of a data record. when it starts doing non-data things I raise an eyebrow. I would certainly wonder if I needed url_helpers in a data record model.

I would be in the market for another PORO. Dumb question can some sort of link component be used??

Reply:
Yes, you are right, I just needed an extra push!

I've gone with a presenter for now, since that's the pattern BoPs uses elsewhere. Would love to discuss a potential link component though!